src/faq.html

```diff
@@ -61,6 +61,17 @@
           <li>Reboot your system.</li>
           <li>Try again.</li>
         </ol> </p>
+  - title: >
+      I'm trying to add a Zigbee device using ZHA, but Home Assistant doesn't find it. What can I do?
+    content: >
+      <p>
+        <ol>
+          <li>Check if there is a <a href="/connectivity/">connectivity issue</a>.</li>
+          <li>Make sure the device is in pairing mode. To find out how to get your device into pairing mode, check the device documentation.</li>
```

Comment:
We could link zigbee.blakadder.com here, since for some devices how to get them into pairing mode is documented there.

```suggestion
          <li>Make sure the device is in pairing mode. To find out how to get your device into pairing mode, check the device documentation. For some devices pairing mode instructions are also documented on <a href="https://zigbee.blakadder.com">zigbee.blakadder.com</a></li>
```